app/components/header_component/view.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if @current_user %>
+  <%= govuk_header navigation_classes: ["govuk-header__navigation--end"] do |header|
+    header.product_name(name: "Forms")
```

Comment:
It could be nice to have the name in the locales rather than here

Reply:
Good spot - have updated!